src/components/Time.jsx

```diff
@@ -43,6 +65,12 @@ class Time extends React.Component {
         return TimeUtils.timeAgo(this.props.time, config)
     }
 
+    _refresh() {
+        this.refresh_timeout = setTimeout(() => {
+            this.setState(() => this.state)
```

Comment:
```suggestion
            this.forceUpdate()
```

Reply:
I don't like using `forceUpdate` since it skips `shouldComponentUpdate`. Seems like a bit of an anti-pattern. In any case, I leave it the final call to you guys. @henningmu @gnapse.

Reply:
If this component had a `shouldComponentUpdate`, which it does not, would you want it to prevent this update? I don't think that's the case. You want this component to refresh at the given interval no matter what.

I don't like to use it in general because it is a signal that the component's output depends on something that's not part of its props or state (meaning is not pure), but in this case this component isn't anyway.

But I too have no hard feelings about it. It's only that reading `setState(this.state)` made me think for a couple of seconds, while `forceUpdate` makes it more explicit the intention.

Reply:
Agreed. `forceUpdate` represents better what `refresh` does, a forced update that doesn't relate to state change. 👍